app/frontend/components/pages/dashboard/walletOperationStatus.tsx

```diff
@@ -24,7 +24,21 @@ export const WalletOperationStatusType = () => {
   }))
   switch (walletOperationStatusType) {
     case 'txPending':
-      return <PinkSpinner message={'Transaction pending...'} />
+      return (
+        <Fragment>
+          <PinkSpinner message={'Transaction pending...'} />
+          <p className="loading-helper-text">
+            Taking too long? Read why{' '}
+            <a
+              href="https://github.com/vacuumlabs/adalite/wiki/Troubleshooting#my-transaction-has-been-pending-for-a-long-time--what-should-i-do"
+              target="_blank"
+            >
+              here
+            </a>
```

Comment:
Just a suggestion, If we wanted to make it cool, we could start a `timer` on `useEffect` and show the message lets say after 10 seconds.